docs/how-to/tips.md

```diff
@@ -0,0 +1,19 @@
+# Snippets and Synchronization
+Imagine you have a page with related tags added via snippets, and you want to query all pages with a specific tag.
```

Comment:
could you add a code snippet illustrating this?
it will help people contextualize the scenario

Reply:
I'm not sure what you mean. I added a code snippet at the bottom ?

Reply:
the tags as snippets, and how they are added to the page

Reply:
added full example, hope this helps